### DISCUSSION TITLE
posts/learningReact

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#11] [General]
# posts/learningReact

http://localhost:3000/posts/learningReact

**Comment by whexy** [comment]
一年后回头看，这篇文章写的真好，是我现在写不出来的水平。而且当时自己的很多思路和想法，再次读到之后也很惊喜。
文章写作时笔者不知道 React hooks，还在用类组件。如果有读者看到这里，请知悉：可以用 `const SomeComponent = () => { return (...) }` 更简单地创建一个组件。
文中提到的很多状态、生命周期的概念，也可以用 `useState()`，`useEffect()` 等 hooks 代替。
